build/__main__.py

```diff
@@ -80,6 +90,9 @@ def main(cli_args):  # type: (List[str]) -> None
     parser.add_argument('--skip-dependencies', '-x',
                         action='store_true',
                         help='does not check for the dependencies')
+    parser.add_argument('--no-isolation', '-n',
```

Comment:
The ``-n`` feels hard to understand, considering would be used rarely manually should we just provide ``--no-isolation`` and no short form, to remove confusion on the caller side?

Reply:
It would be rarely used by users but a lot of packagers will still need to use it manually. A quick query on the arch repos shows that I maintain ~90 python packages, that means I will have to write `--no-isolation` a lot when migrating from `setup.py` calls. Anyway, I think exposing `-n` to users is a fair price, anyone can still use `--help` to find out what it is. Would you be okay keeping it, or is it something you feel strongly about?

Reply:
I would have thought machines/scripts  to invoke this build not you manually. No strong feelings.

Reply:
I usually start from a template with the commands already there, but not always. I think a few people would be against adding macros that abstract the commands because, well, they abstract the commands. And we can't be fully automatic either, we prefer to get sources from github than PyPI, which is not as easy to automate and a lot of the time people would still need to do manually steps.